src/config.zig

```diff
@@ -69,6 +69,14 @@ pub const Config = struct {
 
     cluster: ConfigCluster,
     process: ConfigProcess,
+
+    /// Returns true if the configuration is intended for "production".
+    /// Intended solely for extra sanity-checks: all meaningful decisions should be driven by
+    /// specific fields of the config.
+    pub fn is_production(config: *const Config) bool {
+        _ = config;
+        return build_options.config_base == .production;
```

Comment:
I originally did `config.process.direct_io` thing here, but then realized I can actually cheat and look at the build options directly!